### PR TITLE
Add #pragma once to all header files

### PR DESCRIPTION
**Pull request by jumaffre**

### CODE REVIEW

**Comment by cimetrics** [comment]
pragma_once@[3616 aka 20200102.2](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3616&view=results) vs master ewma over 20 builds from [3489](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3489&view=results) to [3614](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3614&view=results)
images

File: src/consensus/pbft/libbyz/Checkpoint.h [hunk 1]
```diff
@@ -2,20 +2,21 @@
 // Copyright (c) 1999 Miguel Castro, Barbara Liskov.
 // Copyright (c) 2000, 2001 Miguel Castro, Rodrigo Rodrigues, Barbara Liskov.
 // Licensed under the MIT license.
+#pragma once
 
 #ifndef _Checkpoint_h
-#define _Checkpoint_h 1
+#  define _Checkpoint_h 1
```
Comment (achamayou) [inline src/consensus/pbft/libbyz/Checkpoint.h]: Can be removed now?

File: src/consensus/pbft/libbyz/Stable_estimator.h [hunk 2]
```diff
@@ -2,11 +2,12 @@
 // Copyright (c) 1999 Miguel Castro, Barbara Liskov.
 // Copyright (c) 2000, 2001 Miguel Castro, Rodrigo Rodrigues, Barbara Liskov.
 // Licensed under the MIT license.
+#pragma once
 
 #ifndef _Stable_estimator_h
-#define _Stable_estimator_h 1
+#  define _Stable_estimator_h 1
```
Comment (achamayou) [inline src/consensus/pbft/libbyz/Stable_estimator.h]: Ditto

**Comment by cimetrics** [comment]
pragma_once@[3620 aka 20200102.4](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3620&view=results) vs master ewma over 20 builds from [3493](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3493&view=results) to [3618](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3618&view=results)
images

**Comment by cimetrics** [comment]
pragma_once@[3622 aka 20200102.5](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3622&view=results) vs master ewma over 20 builds from [3493](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3493&view=results) to [3618](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3618&view=results)
images

**Pull request merged**